test/integration/connect/utils.py

```diff
@@ -0,0 +1,113 @@
+
+
+def get_commands():
+    common_commands = [
+        # YAML output
+
+        # valid commands
+        Command('return 666', yaml_output='666'),
+        Command('777', yaml_output='777'),
+        # multiline statement
+        Command('if\ntrue\nthen\nreturn 999\nend', yaml_output='999'),
+        # undefined variable
+        Command('kek', yaml_output='error: \'[string "return kek "]:1: variable \'\'kek\'\' is not declared\''),
+        # syntax error
+        Command('if+1', yaml_output='error: \'[string "if+1 "]:1: unexpected symbol near \'\'+\'\'\''),
+    ]
+
+    set_output_error_commands = [
+        Command(
+            '\\set output lua',
+            yaml_output='error: \'[string "\\set output lua "]:1: unexpected symbol near \'\'\\\'\'\''
+        ),
+        Command('return 666', yaml_output='666'),
+
+        Command(
+            '\\set output yaml',
+            yaml_output='error: \'[string "\\set output yaml "]:1: unexpected symbol near \'\'\\\'\'\''
+        ),
+        Command('return 666', yaml_output='666'),
+    ]
+
+    set_output_commands = [
+        # Lua output
+        Command('\\set output lua', lua_output='true'),
+
+        # valid commands
+        Command('return 666', lua_output='666'),
+        Command('777', lua_output='777'),
+        # multiline statement
+        Command('if\ntrue\nthen\nreturn 999\nend', lua_output='999'),
+
+        # YAML output again
+        Command('\\set output yaml', yaml_output='true'),
+        Command('return 666', yaml_output='666'),
+    ]
+
+    commands = common_commands
+
+    if tarantool_short_version().startswith('1.10'):
+        commands.extend(set_output_error_commands)
+    else:
+        commands.extend(set_output_commands)
+
+    return commands
+
+
+def run_commands_in_pipe(project, cmd, commands, exp_connect):
+    process = subprocess.Popen(
+        cmd,
+        cwd=project.path,
+        stderr=subprocess.STDOUT,
+        stdout=subprocess.PIPE,
+        stdin=subprocess.PIPE,
+    )
+
+    user_input = '\n'.join([c.command for c in commands]) + '\n'
+
+    out, _ = process.communicate(user_input.encode('utf-8'))
+    out = out.decode('utf-8')
+
+    print(out)
+
+    out_lines = out.split('\n', maxsplit=1)
+    connected_line, commands_output = out_lines
+
+    assert connected_line == 'connected to %s' % exp_connect
+
+    # now I have no idea how to read commands prompt using subprocess
+    exp_output = '\n'.join(c.exp_output for c in commands)+'\n'
+    assert commands_output == exp_output
+
+
+def send_commands_to_pipe(project, cmd, exp_connect):
+    run_commands_in_pipe(project, cmd, get_commands(), exp_connect)
+
+
+def assert_error(project, cmd, errmsg):
+    process = subprocess.Popen(
+        cmd,
+        cwd=project.path,
+        stderr=subprocess.STDOUT,
+        stdout=subprocess.PIPE,
+        stdin=subprocess.PIPE,
+    )
+
+    output, _ = process.communicate()
+    output = output.decode('utf-8')
+    print(output)
```

Comment:
Do you really need print here?

Reply:
`assert process.returncode == 1, 'Got: ' + output`

Reply:
Yea, I would like this output to be printed anyway. I often use it for local tesing.